api-reference/10 UI Components/dxDropDownBox/1 Configuration/fieldTemplate.md

```diff
@@ -97,12 +99,72 @@ In the following code, the **fieldTemplate** is used to stylize the text field w
         </div>
     </div>
 
-    <!--CSS-->
+    <!-- tab: style.css -->
+    .custom-item {
+        font-size: 16px;
+    }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDropDownBox...>
+            <template #field="{value}">
+                <div class="custom-item">
+                    <DxTextBox
+                        :value="value"
+                        :read-only="true"
+                    />
+                </div>
+            </template>
+        </DxDropDownBox>
+    </template>
+
+    <script>
+        import 'devextreme/dist/css/dx.common.css';
+        import 'devextreme/dist/css/dx.light.css';
+
+        import DxDropDownBox from 'devextreme-vue/drop-down-box';
+        import DxTextBox from 'devextreme-vue/text-box';
```

Comment:
- Align imports with the `<script>` tag.
- The declaration of exported components, data, and methods is missing.